br/pkg/lightning/config/config.go

```diff
@@ -846,7 +851,7 @@ func (cfg *Config) Adjust(ctx context.Context) error {
 	}
 
 	if len(cfg.Mydumper.DataCharacterSet) == 0 {
-		cfg.Mydumper.DataCharacterSet = defaultCSVDataCharacterSet
+		cfg.Mydumper.DataCharacterSet = DefaultCSVDataCharacterSet
```

Comment:
Seems in check only mode, most of the config fields are useless, do we still need to validate them? E.g, tidb.host, pd-addr, sorted-kv-dir..

Reply:
we need `tidb.host` to extract ddl from tidb. some config may not needed, but i think it's fine to read/validate them.